distribution/lib/Standard/AWS/0.0.0-dev/src/S3/S3_File.enso

```diff
@@ -616,3 +616,11 @@ translate_file_errors related_file result =
         s3_path = S3_Path.Value error.bucket error.key
         s3_file = S3_File.Value s3_path related_file.credentials
         Error.throw (File_Error.Not_Found s3_file)
+
+type S3_File_Comparator
```

Comment:
This type should probably be marked as `PRIVATE` as we don't want it to show up in CB.